avahi-core/domain-util.c

```diff
@@ -66,10 +69,10 @@ static int load_lsb_distrib_id(char *ret_s, size_t size) {
         if (!fgets(ln, sizeof(ln), f))
             break;
 
-        if (strncmp(ln, "DISTRIB_ID=", 11))
+        if (strncmp(ln, field, strlen(field)))
```

Comment:
Should be compute strlen(field) only once per function call, not twice per line? Not sure optimization can guess it won't change, but wouldn't hurt if we did that explicitly once just after file were opened. It is no longer literal, but it will never change.